### ISSUE TITLE
Enable Marketplace Page Settings data before submitting to the Marketplace

### ISSUE THREAD

**Issue by johnsicili** [webstudio-is/webstudio#4429]
Currently you have to submit the Project to the Marketplace to expose the per-page Marketplace settings.

This is counterintuitive because those page settings need to be filled out before it's accepted in the Marketplace.

Expectation is to have all fields that need to be set before submitting for review.

<img width="489" alt="image" src="https://github.com/user-attachments/assets/5acfad83-6e5f-45e0-a91f-aa11d601bb2d">

**Comment by TrySound** [comment]
Moreover the project should appear as "personal marketplace product" or "account marketplace product"

**Comment by kof** [comment]
Agree. The reason its hidden was to avoid showing stuff in page settings that isn't used unless its submitted to the marketplace. This makes page settings too busy. 

Problem though that before I submit I should be able to go and check the page settings for the marketplace.

I suggest we show the marketplace page settings, but only the upper part, initially, only when user turns switch on, then we show the rest, this way its not as much space

<img width="454" alt="image" src="https://github.com/user-attachments/assets/5d89c871-d4cb-425b-8795-82bd3e915a87">

**Comment by johnsicili** [comment]
I agree that hiding it for people that don't need it is good.

Maybe we have a project-wide toggle in Project Settings that's like "Enable Marketplace Mode" or something so it's out of the way and with the other Marketplace settings.

**Comment by kof** [comment]
Well marketplace settings is the project-wide toggle. It's just currently anchored with submission for review. I think I pointed out to @TrySound initially that the flow is not logical, but we decided to try. 

The option I proposed at the time was to have a first checkbox in the marketplace settings that would turn that marketplace mode on before you see any marketplace settings, only after that one would see both settings in the project settings marketplace and page settings.

The UX here is also challenging, but we need to revisit it for sure. Something along the lines of a toggle that does nothing but turning on the settings would already solve the problem.
